docs/integrations/containers-orchestration/vmware-tanzu-application-service.md

```diff
@@ -0,0 +1,110 @@
+1. **Errands**. Leave settings as default.
+1. **Resource Config**. For both deploy-all and delete-all, use VM Type setting the Automatic Micro.
+1. **Stemcell**. Ensure the proper stemcell is available.
+1. Return to the Ops Manager Installation Dashboard.
+1. Click **Apply Changes**. The installation can take a few minutes to complete. You will see a message confirming your changes have been applied.
+
+#### Using Sumo Logic Nozzle for VMware Tanzu
+
+This section describes how to use Sumo Logic Nozzle for VMware Tanzu. After installation, the Sumo Logic Nozzle is available in the Apps Manager.
+
+1. From the **system** drop-down menu, select **sumo-logic-nozzle-org**.
+1. Under Apps, select the name of the build you want. For example, **sumo-logic-nozzle-0.1.0-build3**. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/selectnozzle.png')} alt="Select Nozzle" style={{border: '1px solid gray'}} width="800" />
+1. Push the **Start** button to begin running the nozzle.<br/> <img src={useBaseUrl('img/integrations/containers-orchestration/selectplay.png')} alt="Select play" style={{border: '1px solid gray'}} width="400" />
+1. Cloud Foundry will provision the VMs and begin running the nozzle service.
+1. Verify Sumo Logic Nozzle for VMware Tanzu tile is installed and started running the nozzle service by [running searches](/docs/search/search-cheat-sheets/general-search-examples/). If this is your first time using Sumo Logic, see [Getting Started with Search](/docs/search/get-started-with-search/).
+
+:::info
+You can also create [dashboards](/docs/dashboards/) and [alerts](/docs/alerts/).
+:::
+
+:::note
+- For detailed information about changelog, refer to the [Changelog](https://github.com/SumoLogic/sumologic-cloudfoundry-nozzle/blob/master/CHANGELOG.md) file.
+- For support, contact [VMware Tanzu Feedback](mailto:pivotal-cf-feedback@pivotal.io) list or [Sumo Logic Support](https://support.sumologic.com/).
+:::
+
+## Troubleshooting
+
+To troubleshoot the Sumo Logic Nozzle integration with VMware Tanzu:
+1. Log in to **Apps Manager**.
+1. Navigate to **Home** > **ORG** `<sumo-logic-nozzle-org>` > **SPACE** `<sumo-logic-nozzle-space>`.
+2. Click on **Name**. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/Troubleshooting.png')} alt="Troubleshooting" style={{border: '1px solid gray'}} width="800" />
+3. The app page appears. Click **Logs**. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/TroubleshootingApp.png')} alt="Troubleshooting app" style={{border: '1px solid gray'}} width="800" />
+4. On the **Logs** page it will show you logs related to the Sumo Logic Nozzle integration. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/TroubleshootingLogs.png')} alt="Troubleshooting logs" style={{border: '1px solid gray'}} width="800" />
```

Comment:
```suggestion
## Troubleshooting

To troubleshoot the Sumo Logic Nozzle integration with VMware Tanzu:
1. Log in to **Apps Manager**.
1. Navigate to **Home** > **ORG** `<sumo-logic-nozzle-org>` > **SPACE** `<sumo-logic-nozzle-space>`.
2. Click on **Name**. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/Troubleshooting.png')} alt="Troubleshooting" style={{border: '1px solid gray'}} width="800" />
3. The app page appears. Click **Logs**. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/TroubleshootingApp.png')} alt="Troubleshooting app" style={{border: '1px solid gray'}} width="800" />
4. On the **Logs** page it will show you logs related to the Sumo Logic Nozzle integration. <br/><img src={useBaseUrl('img/integrations/containers-orchestration/TroubleshootingLogs.png')} alt="Troubleshooting logs" style={{border: '1px solid gray'}} width="800" />

## More information

- For detailed version information, refer to the [changelog](https://github.com/SumoLogic/sumologic-cloudfoundry-nozzle/blob/master/CHANGELOG.md) file.
- For support, contact [VMware Tanzu Feedback](mailto:pivotal-cf-feedback@pivotal.io) list or [Sumo Logic Support](https://support.sumologic.com).
```

Reply:
I didn't change the Troubleshooting section; just moved the two bullets above that to the end under a "more information" section.